purchase_packaging_level_qty/models/purchase_order.py

```diff
@@ -0,0 +1,53 @@
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    total_transport_qty = fields.Char(
+        compute="_compute_total_transport_packaging_qty",
+    )
+
+    def _get_purchase_transport_packaging_level(self):
+        return self.env.company.purchase_packaging_level_id
+
+    @api.depends("order_line.product_uom_qty", "order_line.product_id")
+    def _compute_total_transport_packaging_qty(self):
+        for order in self:
+            plevel = order._get_purchase_transport_packaging_level()
+            if plevel:
+                total_transport_qty = sum(order.mapped("order_line.transport_qty"))
+                order.total_transport_qty = "{} {}".format(
+                    total_transport_qty,
+                    plevel.code or "",
+                ).strip()
+            else:
+                order.total_transport_qty = False
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    transport_qty = fields.Float(
+        compute="_compute_line_transport_qty",
+        digits="Product Unit of Measure",
+    )
+
+    def _get_transport_packaging_qty(self):
+        plevel = self.order_id._get_purchase_transport_packaging_level()
+        product_packaging = self.product_id.packaging_ids.filtered(
+            lambda self: self.packaging_level_id == plevel
+        )
+        if product_packaging:
+            return sum(product_packaging.mapped("qty"))
```

Comment:
Take the biggest one if there are multiple
```suggestion
            return max(product_packaging.mapped("qty"))
```